src/codegen/properties/attribute.py

```diff
@@ -0,0 +1,281 @@
+class Attribute():
+    
+    def __init__(self, id, attribute_metadata):
+        self._id  = id
+        self._is_enum = False
+        self._access = attribute_metadata["access"]
+        self._name = attribute_metadata["name"].lower()
+        self._resettable = attribute_metadata["resettable"]
+        self._type = attribute_metadata["type"]
+        self._ctypes_data_type = attribute_metadata["ctypes_data_type"]
+        self._python_data_type = attribute_metadata["python_data_type"]
+        self._description = attribute_metadata["description"]
+        self._has_explicit_read_buffer_size = attribute_metadata["has_explicit_read_buffer_size"]
+        self._bitfield_enum = attribute_metadata["bitfield_enum"]
+        self._is_list = attribute_metadata["is_list"]
+        self._calling_convention = attribute_metadata["calling_convention"]
+        self._c_function_name = attribute_metadata["c_function_name"]
+        if("is_object" in attribute_metadata): 
+            self._is_object = attribute_metadata["is_object"]
```

Comment:
I would expect `_is_object` to be set to False in the else case. You could do this:

```python
self._is_object = attribute_metadata.get("is_object", False)
```

Reply:
As suggested, used `dict.get` to get the value out of a dictionary and updated all instances where it has been used.